### PR TITLE
Version 1.5.0

### PR DESCRIPTION
**Pull request by Kalissaac**
Had to cherry-pick changes from `dev`, since we have also been working on slash commands there.

## BREAKING CHANGES
* Automatic team switching has been disabled due to the numerous bugs it caused with the embeds. It will be re-enabled at a later date.

## Added Features
* Match creation prompts are now dynamically created based on values set in `constants.js`
* Added `v!match refresh` command to refresh stuck embeds

## Bug Fixes
* `v!match edit` now supports natural language date parsing
* Username validation now supports special characters

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -68,25 +68,27 @@ module.exports = exports = {
 
   GAME_MODES: ['standard', 'spike rush', 'deathmatch'],
 
-  AFFIRMATIVE_WORDS: ['yes', 'yeah', 'sure', 'true', '1', 'one', 'si', 'yea', 'ok', 'okay', 'mhm', 'k', 'yah', 'on', 'why not', 'alright', 'aight', 'affirmative'],
+  AFFIRMATIVE_WORDS: ['yes', 'yeah', 'yea', 'ye', 'yah', 'sure', 'true', '1', 'one', 'on', 'si', 'ok', 'okay', 'k', 'mhm', 'why not', 'alright', 'aight', 'affirmative'],
```
Comment (MomoMiles) [inline src/constants.js:71]: Add yeet

File: src/constants.js [hunk 2]
```diff
@@ -68,25 +68,27 @@ module.exports = exports = {
...
 
   MAX_TEAM_COUNT: 5, // maximum amount of players allowed on one team
 
   userRegistrationSteps: [
     ['1. Valorant Username', 'What is your FULL Valorant username? (including tag, e.g. `Username#NA1`)'],
-    ['2. Valorant Rank', 'What rank are you in Valorant? If you don\'t have a rank, go with "Iron 1"'],
+    ['2. Valorant Rank', 'What rank are you in Valorant? If you don\'t have a rank, go with "Iron 1".'],
```
Comment (MomoMiles) [inline src/constants.js:77]: We should also allow N/A or remove the rank feature all together idk if anyone uses it

**Review by MomoMiles — APPROVED** [review]
Looks good

File: src/constants.js (hunk 2 above)
Comment (Kalissaac) [reply src/constants.js:77]: People do use the rank feature for the rank roles. Adding N/A require refactoring our rank evaluation when people join a match, so I think we can leave that for later.

**Pull request merged**